src/main/java/io/appform/functionmetrics/TraceManager.java

```diff
@@ -0,0 +1,61 @@
+package io.appform.functionmetrics;
+
+import com.google.common.base.Strings;
+import io.opentracing.Scope;
+import io.opentracing.Span;
+import io.opentracing.util.GlobalTracer;
+
+/**
+ * Created by kanika.khetawat on 03/02/21
+ */
+public class TraceManager {
+
+    private Span span;
+    private Scope scope;
+
+    void startSpan(final String methodName,
+                   final String className,
+                   final String parameterString) {
+        Span parentSpan = GlobalTracer.get().activeSpan();
+        if (parentSpan == null) {
+            return;
+        }
+        span = GlobalTracer.get().buildSpan("method")
```

Comment:
Check for effficiency

Reply:
getActiveSpan, buildSpan, activateSpan... all combined runs in 350ms for 1 million iterations.